omnigibson/utils/asset_conversion_utils.py

```diff
@@ -282,6 +282,13 @@ def _set_mtl_opacity(mtl_prim, texture):
     lazy.omni.usd.create_material_input(mtl_prim, mtl, texture, lazy.pxr.Sdf.ValueTypeNames.Asset)
     lazy.omni.usd.create_material_input(mtl_prim, "enable_opacity", True, lazy.pxr.Sdf.ValueTypeNames.Bool)
     lazy.omni.usd.create_material_input(mtl_prim, "enable_opacity_texture", True, lazy.pxr.Sdf.ValueTypeNames.Bool)
+
+    # Set the opacity to use the alpha channel for its mono-channel value.
+    # This defaults to some other value, which takes opaque black channels in the
+    # image to be fully transparent. This is not what we want.
+    lazy.omni.usd.create_material_input(mtl_prim, "opacity_mode", 0, lazy.pxr.Sdf.ValueTypeNames.Int)
+    lazy.omni.usd.create_material_input(mtl_prim, "opacity_threshold", 0.1, lazy.pxr.Sdf.ValueTypeNames.Float)
```

Comment:
Why is this done twice in the code base? Are both needed?

Reply:
This is the actually needed one (for the pipeline). The other one is a temporary hotfix that I'll remove after the next export.